Modules/itertoolsmodule.c

```diff
@@ -1739,7 +1739,7 @@ static PyMethodDef starmap_methods[] = {
 };
 
 PyDoc_STRVAR(starmap_doc,
-"starmap(function, sequence) --> starmap object\n\
+"starmap(function, iterable) --> starmap object\n\
```

Comment:
Maybe also include another "parameter": `, *` after "iterable" because both function and iterable are positional-only parameters.

However I'm not too familiar with the itertools documentation, so maybe wait for someone else to chime in before you actually change that. Not worth breaking the consistency in the documentation for that.